### DISCUSSION TITLE
connection to the AP failed

### DISCUSSION THREAD

**Discussion by kslslwk** [jomjol/AI-on-the-edge-device#1344] [Q&A]
Hello, I have a problem about wifi.I can't connect wifi with my esp32cam.
I'm not quite sure how the problem arises.
I hope to get your help.Thank you.
![0221114132008](https://user-images.githubusercontent.com/51841292/201582251-6f5f20db-6b1c-4f37-ab09-87b7a848ea27.jpg)

**Comment by caco3** [comment]
Could it bee that the WIFI is too weak?

Or the SSID/password are wrong?

**Comment by eshigarov** [comment]
> Hello, I have a problem about wifi.I can't connect wifi with my esp32cam. I'm not quite sure how the problem arises. I hope to get your help.Thank you. ![0221114132008](https://user-images.githubusercontent.com/51841292/201582251-6f5f20db-6b1c-4f37-ab09-87b7a848ea27.jpg)

Hi! Have you solved it? I faced exactly the same. SSID and pass are correct

**Reply by caco3** [reply]
Do you have an external antenna on your esp?
maybe the accesspoint is too far away?
Or it does not provide the features the ESP needs (eg. 2.4 GHz and standard auth settings).

**Reply by eshigarov** [reply]
> Do you have an external antenna on your esp? maybe the accesspoint is too far away? Or it does not provide the features the ESP needs (eg. **2.4 GHz** and standard auth settings).
Solved, seemed to be 2.4 GHz, thanks
